storage/client/queue/src/main/java/com/azure/storage/common/credentials/SharedKeyCredential.java

```diff
@@ -98,9 +98,11 @@ public String generateAuthorizationHeader(URL requestURL, String httpMethod, Map
      *
      * @param stringToSign The UTF-8-encoded string to sign.
      * @return A {@code String} that contains the HMAC-SHA256-encoded signature.
-     * @throws InvalidKeyException If the accountKey is not a valid Base64-encoded string.
+     * @throws RuntimeException for one of the following cases:
+     *                              - If the HMAC-SHA256 signature for {@code sharedKeyCredentials} fails to generate.
+     *                              - If the an invalid key has been given to the client.
```

Comment:
Generate the JavaDoc and review this - it won't look good most probably.

Reply:
The sharedKey is in common. Alan and I prefer to have the queue and file merged in first, and then have another PR for common specifically.